draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -507,7 +507,9 @@ The CHOICE allows three representations of the private key:
    derived from the seed.
 
 3. The both format contains both the seed and expanded key, allowing for
-   validation that the expanded key was correctly derived from the seed.
+   for interoperability with a broad range of consumers, some that may
+   want to use and retain the seed, others that may only support expanded
+   keys.
```

Comment:
Ah you were talking about this. I think this captures what you are asking for.

Reply:
I am good with this, but I would suggest to remove the "broad range of consumers". I don't think it adds much. Maybe 

> for interoperability with consumers that may want to use and retain the seed, and others that may only support expanded keys. 

I am not married to it. I would live with the text as is.

Reply:
The result ended up a bit mangled grammatically. :-(